script/lint.js

```diff
@@ -69,12 +71,20 @@ const LINTERS = [{
   roots: ['shell'],
   test: filename => filename.endsWith('.cc') || (filename.endsWith('.h') && !isObjCHeader(filename)),
   run: (opts, filenames) => {
-    if (opts.fix) {
-      spawnAndCheckExitCode('python', ['script/run-clang-format.py', '--fix', ...filenames]);
-    } else {
-      spawnAndCheckExitCode('python', ['script/run-clang-format.py', ...filenames]);
+    const chunkedFilenames = chunkFilenames(filenames);
+
+    filenames = chunkedFilenames.shift();
+
+    while (filenames) {
+      if (opts.fix) {
+        spawnAndCheckExitCode('python', ['script/run-clang-format.py', '--fix', ...filenames]);
+      } else {
+        spawnAndCheckExitCode('python', ['script/run-clang-format.py', ...filenames]);
+      }
+      cpplint(filenames);
+
+      filenames = chunkedFilenames.shift();
```

Comment:
Slightly clearer?

```suggestion
    for (const chunk of chunkFilenames(filenames))
    {
      if (opts.fix) {
        spawnAndCheckExitCode('python', ['script/run-clang-format.py', '--fix', ...chunk]);
      } else {
        spawnAndCheckExitCode('python', ['script/run-clang-format.py', ...chunk]);
      }
      cpplint(filenames);
```

Reply:
Agreed. Not sure why I wrote it the other way to begin with. 😅

Reply:
Ah, now I remember. The usage in the `clang-tidy` script is using it like a queue for async workers so it needs to only pull out one chunk at a time for each worker, which isn't a need here.